aws/logs_monitoring/release.sh

```diff
@@ -41,8 +41,18 @@ if [ "$ACCOUNT" = "prod" ]; then
     BUCKET="datadog-cloudformation-template"
 fi
 
+function awe-login() {
```

Comment:
This function doesn't work on my machine and I can't figure out why.

```
awe-login aws sts get-caller-identity
Enter token for arn:aws:iam::464622532012:mfa/chris.agocs: 629648
aws-vault: error: exec: Error execing process: exec: "aws sts get-caller-identity": executable file not found in $PATH
```

I'd feel more comfortable if we could defensively code around whatever's going on here, or if we used aws-vault directly

Reply:
Yeah, unfortunately we can't use aws-vault directly on release.sh, because it aws-vault doesn't support nested sessions, (I tried unsetting the relevant environment variables, but that broke things even more). Can you call 'aws sts get-caller-identity' directly from bash/zsh. It's weird that it's not in your path.

Reply:
```suggestion
function awe-login() {
    cfg=( "$@" )
    echo "${cfg[@]}"
    shift
    if [ "$ACCOUNT" = "prod" ] ; then
        # shellcheck disable=SC2068
        aws-vault exec prod-engineering --  ${cfg[@]}
    else
        # shellcheck disable=SC2068
        aws-vault exec sandbox-account-admin --  ${cfg[@]}
    fi
}
```

Reply:
Here, this works

```
function awe-login() {
    cfg=( "$@" )
    echo "${cfg[@]}"
    shift
    if [ "$ACCOUNT" = "prod" ] ; then
        # shellcheck disable=SC2068
        aws-vault exec prod-engineering --  ${cfg[@]}
    else
        # shellcheck disable=SC2068
        aws-vault exec sandbox-account-admin --  ${cfg[@]}
    fi
}
```

Reply:
I've updated the function to use this.